account.go

```diff
@@ -49,17 +51,19 @@ func IsConnectedToRoutingNode() bool {
 /*
 createChannel is responsible for creating a new channel
 */
-func createChannel(pubkey string) error {
+func createChannel(pubkey string) {
 	c := breezservice.NewFundManagerClient(breezClientConnection)
 	ctx, cancel := context.WithTimeout(context.Background(), endpointTimeout*time.Second)
 	defer cancel()
-	_, err := c.OpenChannel(ctx, &breezservice.OpenChannelRequest{PubKey: pubkey})
-	if err != nil {
-		log.Errorf("Error in openChannel: %v", err)
-		return err
+	for {
+		_, err := c.OpenChannel(ctx, &breezservice.OpenChannelRequest{PubKey: pubkey})
+		if err != nil {
+			log.Errorf("Error in openChannel: %v", err)
+			time.Sleep(time.Second * 5)
+			continue
+		}
+		return
```

Comment:
Maybe you should break out of this if connectedToRouting node is 0 as well? So we don't needlessly keep trying when we are disconnected anyway?

Reply:
Agree, nice one!